PayItForward.Data.Services/StoriesService.cs

```diff
@@ -79,6 +79,7 @@ public int CountStories(string subTitle = "", string categoryname = "")
         public IEnumerable<StoryDTO> Stories()
         {
             var storiesFromDb = this.storiesRepo.GetAll()
+                .Where(x => x.IsDeleted == false)
```

Comment:
You may add to the repository GetAll method to return you only the not deleted records and you can add one more method that will return you all items with deleted together, so you can skip this check from everywhere